modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/qpid/ServerConfiguration.java

```diff
@@ -567,6 +567,11 @@ public boolean getSSLOnly() {
                 !(Boolean)AndesConfigurationManager.readValue(AndesConfiguration.TRANSPORTS_AMQP_DEFAULT_CONNECTION_ENABLED);
     }
 
+    public boolean getMQTTSSLOnly() {
+        return (Boolean) AndesConfigurationManager.readValue(AndesConfiguration.TRANSPORTS_MQTT_SSL_CONNECTION_ENABLED) &&
+               !(Boolean)AndesConfigurationManager.readValue(AndesConfiguration.TRANSPORTS_MQTT_DEFAULT_CONNECTION_ENABLED);
```

Comment:
Currently this is how we check if only SSL port enabled in AMQP implementation. https://github.com/pumudu88/andes/blob/fix_mqtt_issues/modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/qpid/ServerConfiguration.java#L565-L568
Will refactor this in more suitable way for both AMQP and MQTT in another iteration.